test/server/cards/02_SHD/upgrades/Foundling.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('Foundling', function () {
+    integration(function () {
+        describe('Foundling', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['foundling', 'mandalorian-warrior'],
+                        groundArena: ['battlefield-marine', 'pyke-sentinel', 'protector-of-the-throne'],
+                    },
+                    player2: {
+                        groundArena: ['snowspeeder']
+                    }
+                });
+            });
+
+            it('\'s ability should give the attached card the Mandalorian trait', function () {
```

Comment:
nit: jasmine adds a space in between the "describe" string and the "it" string. So in this case it would come out as `Foundling 's ability should...` instead of `Foundling's ability should...`. If you move the `\'s` into `describe` it will look correct